### PR TITLE
[zh] sync networking virtual-ips

### PR DESCRIPTION
**Pull request by ktKongTong**
sync with en.

`/reference/networking/virtual-ips.md`

fixed: https://github.com/kubernetes/website/issues/38295

preview: https://deploy-preview-38331--kubernetes-io-main-staging.netlify.app/

### CODE REVIEW

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+## {{% heading "whatsnext" %}}
+ -->
+## {{% heading "whatsnext" %}}
+
+<!-- 
+To learn more about Services,
+read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/).
+ -->
+要了解有关 Service 的更多信息，
+请阅读 [使用 Service 连接应用程序](/zh-cn/docs/concepts/services-networking/connect-applications-service/)。
```
Comment (sftim) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:675]: If you want, skip the redirect and link directly to `/zh-cn/docs/tutorials/services/connect-applications-service/`. This is OK even when the English version still relies on a redirect.

Comment (ktKongTong) [reply content/zh-cn/docs/reference/networking/virtual-ips.md]: I checked and found that  there was no Chinese version of  `/docs/concepts/services-networking/connect-applications-service/`.
Now it has been changed.

**Comment by ktKongTong** [comment]
> temporary reviewed here

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+image-processing workload, running with three replicas. Those replicas are
+fungible&mdash;frontends do not care which backend they use.  While the actual Pods that
+compose the backend set may change, the frontend clients should not need to be aware of that,
+nor should they need to keep track of the set of backends themselves.
+ -->
+
+<a id="example"></a>
+本文档中的一些详细内容参考了这样一个例子：
+运行了 3 个 Pod 副本的无状态图像处理后端工作负载。
+这些副本是可互换的 —— 前端不需要关心它们调用了哪个后端副本。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:94]:
```suggestion
这些副本是可互换的；前端不需要关心它们调用了哪个后端副本。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+- The ConfigMap parameters for the kube-proxy cannot all be validated and verified on startup.
+  For example, if your operating system doesn't allow you to run iptables commands,
+  the standard kernel kube-proxy implementation will not work.
+  Likewise, if you have an operating system which doesn't support `netsh`,
+  it will not run in Windows userspace mode.
+ -->
+
+注意，kube-proxy 会根据不同配置以不同的模式启动。
+
+- kube-proxy 的配置是通过 ConfigMap 完成的，kube-proxy 的 ConfigMap 有效地弃用了 大部分 kube-proxy 标志的行为。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:122]:
```suggestion
- kube-proxy 的配置是通过 ConfigMap 完成的，kube-proxy 的 ConfigMap 有效地弃用了大部分 kube-proxy 标志的行为。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+opens a port (randomly chosen) on the local node. Any connections to this _proxy port_
+are proxied to one of the Service's backend Pods (as reported via
+EndpointSlices). The kube-proxy takes the `sessionAffinity` setting of the Service into
+account when deciding which backend Pod to use.
+ -->
+
+这种（遗留）模式使用 iptables 添加拦截规则，然后使用 kube-proxy 工具执行流量转发。
+kube-proxy 监视 Kubernetes 控制平面对 Service 和 EndpointSlice 对象的增加、修改和删除。
+对于每个 Service，kube-proxy 在本地节点上打开一个端口（随机选择）。
+任何对这个 **代理端口** 的连接都将代理到 Service 的一个后端 Pod（通过 EndpointSlices 报告）。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:149]:
```suggestion
任何对这个**代理端口**的连接都将代理到 Service 的一个后端 Pod（通过 EndpointSlices 报告）。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+kube-proxy 在决定使用哪个后端 Pod 时会考虑 Service 的 `sessionAffinity` 设置。
+
+<!-- 
+The user-space proxy installs iptables rules which capture traffic to the
+Service's `clusterIP` (which is virtual) and `port`. Those rules redirect that traffic
+to the proxy port which proxies the backend Pod.
+ -->
+用户空间代理添加 iptables 规则，这些规则捕获流向 Service 的 `clusterIP`（虚拟的）和 `port` 的流量。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:157]:
```suggestion
用户空间代理添加 iptables 规则，这些规则捕获流向 Service 的 `clusterIP`（虚拟 IP）和 `port` 的流量。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 6]
```diff
@@ -0,0 +1,687 @@
...
+As an example, consider the image processing application described [earlier](#example)
+in the page.
+When the backend Service is created, the Kubernetes control plane assigns a virtual
+IP address, for example 10.0.0.1.  Assuming the Service port is 1234, the
+Service is observed by all of the kube-proxy instances in the cluster.
+When a proxy sees a new Service, it opens a new random port, establishes an
+iptables redirect from the virtual IP address to this new port, and starts accepting
+connections on it.
+ -->
+考虑[前面](#example)描述的图像处理应用的例子。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:184]:
```suggestion
例如，考虑本文[前面](#example)描述的图像处理应用的例子。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 7]
```diff
@@ -0,0 +1,687 @@
...
+当创建后端 Service 时，Kubernetes 控制平面分配一个虚拟 IP 地址，例如 10.0.0.1。
+假设 Service 端口是 1234，那么集群中的所有 kube-proxy 实例都会观察到该 Service。
+当一个 kube-proxy 观察到新 Service 时，它会随机打开一个新端口，建立从虚拟 IP 地址到这个新端口的 iptables 重定向，并开始在其上接受连接。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:187]: 适当换行，例如在中英文交叉，标点符号等
```suggestion
当一个 kube-proxy 观察到新 Service 时，它会随机打开一个新端口，
建立从虚拟 IP 地址到这个新端口的 iptables 重定向，并开始在其上接受连接。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 8]
```diff
@@ -0,0 +1,687 @@
...
+<!-- 
+Using the userspace proxy obscures the source IP address of a packet accessing
+a Service.
+This makes some kinds of network filtering (firewalling) impossible.  The iptables
+proxy mode does not
+obscure in-cluster source IPs, but it does still impact clients coming through
+a load balancer or node-port.
+ -->
+使用用户空间代理会隐藏访问 Service 的数据包的源 IP 地址。
+这使得某些类型的网络过滤（防火墙）变得不可能。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:230]:
```suggestion
这使得某些类型的网络过滤（防火墙）失效。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 9]
```diff
@@ -0,0 +1,687 @@
...
+In this mode, kube-proxy watches the Kubernetes control plane for the addition and
+removal of Service and EndpointSlice objects. For each Service, it installs
+iptables rules, which capture traffic to the Service's `clusterIP` and `port`,
+and redirect that traffic to one of the Service's
+backend sets. For each endpoint, it installs iptables rules which
+select a backend Pod.
+ -->
+在这种模式下，kube-proxy 监视 Kubernetes 控制平面对 Service 和 EndpointSlice 对象的添加和删除。
+对于每个 Service，kube-proxy 会添加 iptables 规则，这些规则捕获流向 Service 的 `clusterIP` 和 `port` 的流量，
+并将这些流量重定向到 Service 后端集合中的一个后端。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:249]:
```suggestion
并将这些流量重定向到 Service 后端集合中的其中之一。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 10]
```diff
@@ -0,0 +1,687 @@
...
+集群中的所有 kube-proxy 实例都会观察到新 Service 的创建。
+
+<!-- 
+When kube-proxy on a node sees a new Service, it installs a series of iptables rules
+which redirect from the virtual IP address to more iptables rules, defined per Service.
+The per-Service rules link to further rules for each backend endpoint, and the per-
+endpoint rules redirect traffic (using destination NAT) to the backends.
+ -->
+当节点上的 kube-proxy 观察到新 Service 时，它会添加一系列 iptables 规则，
+这些规则从虚拟 IP 地址重定向到更多 iptables 规则，每个服务都定义了这些规则。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:316]:
```suggestion
这些规则从虚拟 IP 地址重定向到更多 iptables 规则，每个 Service 都定义了这些规则。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 11]
```diff
@@ -0,0 +1,687 @@
...
+每个服务规则链接到每个后端端点的更多规则，
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:317]:
```suggestion
每个 Service 规则链接到每个后端端点的更多规则，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 12]
```diff
@@ -0,0 +1,687 @@
...
+
+<!-- 
+When a client connects to the Service's virtual IP address the iptables rule kicks in.
+A backend is chosen (either based on session affinity or randomly) and packets are
+redirected to the backend.  Unlike the userspace proxy, packets are never
+copied to userspace, the kube-proxy does not have to be running for the virtual
+IP address to work, and Nodes see traffic arriving from the unaltered client IP
+address.
+ -->
+ 当客户端连接到 Service 的虚拟 IP 地址时，iptables 规则会生效。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:328]:
```suggestion
当客户端连接到 Service 的虚拟 IP 地址时，iptables 规则会生效。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 13]
```diff
@@ -0,0 +1,687 @@
...
+* `lc`: least connection (smallest number of open connections)
+* `dh`: destination hashing
+* `sh`: source hashing
+* `sed`: shortest expected delay
+* `nq`: never queue
+ -->
+
+IPVS 为将流量均衡到后端 Pod 提供了更多选择：
+
+* `rr` ：轮询
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:392]:
```suggestion
* `rr`：轮询
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 14]
```diff
@@ -0,0 +1,687 @@
...
+
+<!-- 
+To run kube-proxy in IPVS mode, you must make IPVS available on
+the node before starting kube-proxy.
+
+When kube-proxy starts in IPVS proxy mode, it verifies whether IPVS
+kernel modules are available. If the IPVS kernel modules are not detected, then kube-proxy
+falls back to running in iptables proxy mode.
+ -->
+要在 IPVS 模式下运行 kube-proxy，必须在启动 kube-proxy 之前在节点上启用 IPVS。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:410]:
```suggestion
要在 IPVS 模式下运行 kube-proxy，必须在启动 kube-proxy 之前确保节点上的 IPVS 可用。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 15]
```diff
@@ -0,0 +1,687 @@
...
+
+<!-- 
+Unlike Pod IP addresses, which actually route to a fixed destination,
+Service IPs are not actually answered by a single host.  Instead, kube-proxy
+uses packet processing logic (such as Linux iptables) to define _virtual_ IP
+addresses which are transparently redirected as needed.
+ -->
+与实际路由到固定目标的 Pod IP 地址不同，Service IP 实际上不是由单个主机回答的。
+相反，kube-proxy 使用数据包处理逻辑（例如 Linux 的 iptables）
+来定义 **虚拟** IP 地址，这些地址会按需被透明重定向。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:478]:
```suggestion
来定义**虚拟** IP 地址，这些地址会按需被透明重定向。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 16]
```diff
@@ -0,0 +1,687 @@
...
+
+<!-- 
+One of the primary philosophies of Kubernetes is that you should not be
+exposed to situations that could cause your actions to fail through no fault
+of your own. For the design of the Service resource, this means not making
+you choose your own port number if that choice might collide with
+someone else's choice.  That is an isolation failure.
+ -->
+Kubernetes 的主要哲学之一是，
+你不应被暴露在可能导致你操作失败但完全不是你的问题的情况下。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:501]:
```suggestion
你不应在可能导致你操作失败但完全不是你的问题的情况下暴露。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 17]
```diff
@@ -0,0 +1,687 @@
...
+CIDR 范围内为每个 Service 分配自己的 IP 地址来实现这一点。
+
+<!-- 
+To ensure each Service receives a unique IP, an internal allocator atomically
+updates a global allocation map in {{< glossary_tooltip term_id="etcd" >}}
+prior to creating each Service. The map object must exist in the registry for
+Services to get IP address assignments, otherwise creations will
+fail with a message indicating an IP address could not be allocated.
+ -->
+
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:523]:
```suggestion
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 18]
```diff
@@ -0,0 +1,687 @@
...
+对于 Service 资源的设计，也就是如果你选择的端口号可能与其他人的选择冲突，
+就不应该让你自己选择端口号。这是一种失败隔离。
+
+<!-- 
+In order to allow you to choose a port number for your Services, we must
+ensure that no two Services can collide. Kubernetes does that by allocating each
+Service its own IP address from within the `service-cluster-ip-range`
+CIDR range that is configured for the API server.
+ -->
+
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:511]:
```suggestion
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 19]
```diff
@@ -0,0 +1,687 @@
...
+<!-- 
+Kubernetes divides the `ClusterIP` range into two bands, based on
+the size of the configured `service-cluster-ip-range` by using the following formula
+`min(max(16, cidrSize / 16), 256)`. That formula paraphrases as _never less than 16 or
+more than 256, with a graduated step function between them_.
+ -->
+Kubernetes 根据配置的 `service-cluster-ip-range` 的大小使用公式
+ `min(max(16, cidrSize / 16), 256)` 将 `ClusterIP` 范围分为两段。
+ 该公式意思是 **中间是一个渐进的阶梯函数，但不小于 16 且不大于 256**。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:555]:
```suggestion
该公式意思是**中间是一个渐进的阶梯函数，但不小于 16 且不大于 256**。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 20]
```diff
@@ -0,0 +1,687 @@
...
+
+<!-- 
+Kubernetes prefers to allocate dynamic IP addresses to Services by choosing from the upper band,
+which means that if you want to assign a specific IP address to a `type: ClusterIP`
+Service, you should manually assign an IP address from the **lower** band. That approach
+reduces the risk of a conflict over allocation.
+ -->
+Kubernetes 优先通过从高段中选择来为 Service 分配动态 IP 地址，
+这意味着如果要将特定 IP 地址分配给 `type: ClusterIP` Service，
+则应手动从 **低**段中分配 IP 地址。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:565]:
```suggestion
则应手动从**低**段中分配 IP 地址。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 21]
```diff
@@ -0,0 +1,687 @@
...
+<!-- 
+On Windows, setting the maximum session sticky time for Services is not supported.
+ -->
+在 Windows 上不支持为 Service 设置最大会话粘性时间。
+{{< /note >}}
+
+<!-- 
+## IP address assignment to Services
+ -->
+## 将 IP 地址分配给 Service
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:468]:
```suggestion
## 将 IP 地址分配给 Service  {#ip-address-assignment-to-services}
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 22]
```diff
@@ -0,0 +1,687 @@
...
+has local endpoints and whether or not all the local endpoints are marked as terminating.
+If there are local endpoints and **all** of those are terminating, then the kube-proxy ignores
+any external traffic policy of `Local`. Instead, whilst the node-local endpoints remain as all
+terminating, the kube-proxy forwards traffic for that Service to healthy endpoints elsewhere,
+as if the external traffic policy were set to `Cluster`.
+ -->
+如果为 kube-proxy 启用了 `ProxyTerminatingEndpoints`
+[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)，
+kube-proxy 会检查节点是否具有本地端点以及是否所有本地端点都标记为终止。
+如果有本地端点并且 **所有** 本地端点都被标记为终止，则 kube-proxy 忽略 `Local` 的任何外部流量策略。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:626]:
```suggestion
如果有本地端点并且**所有**本地端点都被标记为终止，则 kube-proxy 忽略 `Local` 的任何外部流量策略。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 23]
```diff
@@ -0,0 +1,687 @@
...
+
+
+<!--
+This forwarding behavior for terminating endpoints exists to allow external load balancers to
+gracefully drain connections that are backed by `NodePort` Services, even when the health check
+node port starts to fail. Otherwise, traffic can be lost between the time a node is
+still in the node pool of a load balancer and traffic is being dropped during the
+termination period of a pod.
+ -->
+
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:639]:
```suggestion
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md (hunk 7 above)
Comment (ktKongTong) [reply content/zh-cn/docs/reference/networking/virtual-ips.md]: 嗯嗯，编辑时是有意识在换行的，这里是我没注意到，遗漏了🥲👍。

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 24]
```diff
@@ -0,0 +1,685 @@
...
+<!-- 
+Kubernetes divides the `ClusterIP` range into two bands, based on
+the size of the configured `service-cluster-ip-range` by using the following formula
+`min(max(16, cidrSize / 16), 256)`. That formula paraphrases as _never less than 16 or
+more than 256, with a graduated step function between them_.
+ -->
+Kubernetes 根据配置的 `service-cluster-ip-range` 的大小使用公式
+ `min(max(16, cidrSize / 16), 256)` 将 `ClusterIP` 范围分为两段。
+该公式意思是**中间是一个渐进的阶梯函数，但不小于 16 且不大于 256**。
```
Comment (ktKongTong) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:554]:
```suggestion
该公式是一个**不小于 16 且不大于 256 的渐进阶梯函数**。
```

Comment (ktKongTong) [reply content/zh-cn/docs/reference/networking/virtual-ips.md]: 此处原文如下

Kubernetes divides the `ClusterIP` range into two bands, based on
the size of the configured `service-cluster-ip-range` by using the following formula
`min(max(16, cidrSize / 16), 256)`. That formula paraphrases as _never less than 16 or
more than 256, with a graduated step function between them_.

对其斜体部分的翻译我想了三种，目前采取的第二种，请问各位 reviewer 有什么更好的建议吗？
`该公式是一个**渐进阶梯函数，其上界为 256，下界为 16。**`
`该公式是一个**不小于 16 且不大于 256 的渐进阶梯函数**。`
`该公式意思是**中间是一个渐进的阶梯函数，但不小于 16 且不大于 256**。`

Comment (howieyuen) [reply content/zh-cn/docs/reference/networking/virtual-ips.md]: vote for No.2

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 25]
```diff
@@ -0,0 +1,685 @@
...
+has local endpoints and whether or not all the local endpoints are marked as terminating.
+If there are local endpoints and **all** of those are terminating, then the kube-proxy ignores
+any external traffic policy of `Local`. Instead, whilst the node-local endpoints remain as all
+terminating, the kube-proxy forwards traffic for that Service to healthy endpoints elsewhere,
+as if the external traffic policy were set to `Cluster`.
+ -->
+如果为 kube-proxy 启用了 `ProxyTerminatingEndpoints`
+[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)，
+kube-proxy 会检查节点是否具有本地端点以及是否所有本地端点都标记为终止。
+如果有本地端点并且**所有**本地端点都被标记为终止，则 kube-proxy 忽略 `Local` 的任何外部流量策略。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:625]:
```suggestion
如果有本地端点并且**所有**本地端点都被标记为终止，则 kube-proxy 忽略所有取值为 `Local` 的外部流量策略。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 26]
```diff
@@ -0,0 +1,685 @@
...
+相反，当余下的所有本地节点端点处于终止中时，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:626]:
```suggestion
相反，当所有本地节点端点均处于终止中时，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 27]
```diff
@@ -0,0 +1,685 @@
...
+## {{% heading "whatsnext" %}}
+ -->
+## {{% heading "whatsnext" %}}
+
+<!-- 
+To learn more about Services,
+read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/).
+ -->
+要了解有关 Service 的更多信息，
+请阅读 [使用 Service 连接应用](/zh-cn/docs/tutorials/services/connect-applications-service/)。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:674]:
```suggestion
请阅读[使用 Service 连接应用](/zh-cn/docs/tutorials/services/connect-applications-service/)。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 28]
```diff
@@ -0,0 +1,685 @@
...
+<!-- 
+You can also:
+
+* Read about [Services](/docs/concepts/services-networking/service/)
+* Read the [API reference](/docs/reference/kubernetes-api/service-resources/service-v1/) for the Service API 
+-->
+
+也可以：
+* 阅读 [Service](/zh-cn/docs/concepts/services-networking/service/)
+* 阅读 [API 参考](/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1/) 了解更多 Service API
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:685]:
```suggestion
* 阅读 [API 参考](/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1/)进一步了解 Service API
```

**Pull request merged**